src/utils/Dom.ts

```diff
@@ -19,6 +19,15 @@ export interface IOffset {
 export class Dom {
   private static CLASS_NAME_REGEX = /-?[_a-zA-Z]+[_a-zA-Z0-9-]*/g;
   private static ONLY_WHITE_SPACE_REGEX = /^\s*$/;
+  private static forceNativeEvent = false;
+
+  // Force the use of native javascript event.
```

Comment:
Remove comment, and add public jsdoc comment if it needs to be publically documented.

Reply:
I will ask Stephanie for a proper public documentation.